P3826/P3826.md

```diff
@@ -1948,10 +1957,10 @@ subsection as follows:]{.ednote}
 
 > 2. The names `when_all` and `when_all_with_variant` denote customization point objects.
 >    Let `sndrs` be a pack of subexpressions, let `Sndrs` be a pack of the types
->    `decltype((sndrs))...`, [let `env` be a subexpression,]{.add} and let
->    [`CD`]{.rm}[```@_`when-all-domain`_@(env)```]{.add} be the [type]{.rm}[expression]{.add}
+>    `decltype((sndrs))...`, [let `tag` and `env` be subexpressions,]{.add} and let
+>    [`CD`]{.rm}[```@_`when-all-domain`_@(tag, env)```]{.add} be the [type]{.rm}[expression]{.add}
 >    [```common_type_t<decltype(@_`get-domain-early`_@(sndrs))...>```]{.rm}.
->    [```common_type_t<decltype(get_completion_domain<set_value_t>(sndrs, env))...>()```]{.add}.
+>    [```common_type_t<decltype(get_completion_domain<tag>(sndrs, env))...>()```]{.add}.
```

Comment:
I don't think that `common_type_t` works here. The expression is ill-formed if we can't find a common type

Reply:
i think that's ok so long as we handle the case of it being ill-formed at the point we use `when-all-domain(tag, env)`. in `impls-for<when_all_t>::check-types`, we have:

> _Throws_: Any exception thrown as a result of evaluating the _Effects_, or an exception of an unspecified type derived from exception when `CD` is ill-formed.

we need to update this to say `when-all-env(tag, env)` instead of `CD`.